### PR TITLE
Display better error message while parsing JSON

### PR DESCRIPTION
**Pull request by sbimochan**
### Description
Errors are really helpful for devs. Having not known where the issue is not a better DX. Although the code mirror points out where the error is in the line, it has poor visibility and the disabled state in Save button has poor visibility as well.

### Types of Changes
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Screenshots (if appropriate):

Screen Shot 2022-10-25 at 11 28 52 PM

### Known issue
The line number is not accurate. Since it's stringifying everything in one line, the line number is not accurate. Another PR could be to divide by number of double quotes, comma and slash /n s to provide near line number or a better approach.

Another PR could be similar implementation for Create Page by another contributor.

Let's discuss more on thread about this feature.

### CODE REVIEW

**Comment by kritish-dhaubanjar** [comment]
@sbimochan , seems like, `JSON.parse(..)` in chrome yells the position while in `firefox`, the message is helpful with the line number & column. :thinking: 

| Firefox | Chrome |
|-|-|
|image|image|

**Comment by sbimochan** [comment]
> @sbimochan , seems like, `JSON.parse(..)` in chrome yells the position while in `firefox`, the message is helpful with the line number & column. :thinking: 
> 
> 
> 
> | Firefox | Chrome |
> 
> |-|-|
> 
> |image|image|
> 
> 
> 
> 

That's because it's a developer edition firefox. What about normal firefox?

File: app/src/views/items/EditItem.vue [hunk 1]
```diff
@@ -81,9 +84,10 @@ import * as CodeMirror from "codemirror";
 import { linter } from "@codemirror/lint";
 import { oneDark } from "@codemirror/theme-one-dark";
 import { json, jsonParseLinter } from "@codemirror/lang-json";
+import parseJson from "parse-json";
```
Comment (kritish-dhaubanjar) [inline app/src/views/items/EditItem.vue:87]: The result from `parseJson` & catch block of `JSON.parse`  looks similar. Should we use this package? Changes in https://github.com/kritish-dhaubanjar/dynamodb-dashboard/pull/1 looks simpler.

Comment (sbimochan) [reply app/src/views/items/EditItem.vue:87]: What kind of error messages did you try?

**Comment by sbimochan** [comment]
Removed parse json package since there was it's peer dependency issues and the lines and columns didn't work as expected

**Pull request merged**